File: app/src/main/java/com/aplicacion/mypet/Mensaje.java
package com.aplicacion.mypet;

public class Mensaje {
    private String id;
    private String idSender;
    private String idReceiver;
    private String idChat;
    private String Mensaje;
    private long timestamp;
    private boolean visto;

    public Mensaje() {
    }

    public Mensaje(String id, String idSender, String idReceiver, String idChat, String mensaje, long timestamp, boolean visto) {
        this.id = id;
        this.idSender = idSender;
        this.idReceiver = idReceiver;
        this.idChat = idChat;
        Mensaje = mensaje;
        this.timestamp = timestamp;
        this.visto = visto;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getIdSender() {
        return idSender;
    }

    public void setIdSender(String idSender) {
        this.idSender = idSender;
    }

    public String getIdReceiver() {
        return idReceiver;
    }

    public void setIdReceiver(String idReceiver) {
        this.idReceiver = idReceiver;
    }

    public String getIdChat() {
        return idChat;
    }

    public void setIdChat(String idChat) {
        this.idChat = idChat;
    }

    public long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(long timestamp) {
        this.timestamp = timestamp;
    }

    public boolean isVisto() {
        return visto;
    }

    public void setVisto(boolean visto) {
        this.visto = visto;
    }

    public String getMensaje() {
        return Mensaje;
    }

    public void setMensaje(String mensaje) {
        Mensaje = mensaje;
    }
}

File: app/src/main/java/com/aplicacion/mypet/models/Mensaje.kt
package com.aplicacion.mypet.models

class Mensaje {
    var id: String? = null
    var idSender: String? = null
    var idReceiver: String? = null
    var idChat: String? = null
    var mensaje: String? = null
    var timestamp: Long = 0
    var visto = false

    constructor()

    constructor(id: String?, idSender: String?, idReceiver: String?, idChat: String?, mensaje: String, timestamp: Long, visto: Boolean) {
        this.id = id
        this.idSender = idSender
        this.idReceiver = idReceiver
        this.idChat = idChat
        this.mensaje = mensaje
        this.timestamp = timestamp
        this.visto = visto
    }
}